main/saturn-node.js

```diff
@@ -152,12 +155,17 @@ async function start (/** @type {Context} */ ctx) {
     console.log(msg)
     appendToChildLog(msg)
     ctx.recordActivity({ source: 'Saturn', type: 'info', message: msg })
-    Sentry.captureException(new Error(msg), scope => {
+
+    ready = false
+    moduleExitReason = signal || code ? reason : null
+  })
+
+  childProcess.on('close', () => {
```

Comment:
Do we need two `on('close')` listeners? Cannot we add this code to the existing listener on lines 142/145 above?

Reply:
Good call, I merged the two 👍